applications/TrilinosApplication/custom_utilities/trilinos_cutting_iso_app.h

```diff
@@ -268,7 +268,7 @@ class TrilinosCuttingIsosurfaceApplication
 
         for (int index=0; index!=nnodes; ++index)  //now we have to save the nodes!.
         {
-            // KRATOS_WATCH(auxiliar);
```

Comment:
same

Reply:
Done

Reply:
BTW, these two file the code is super duplicated